README.md

```diff
@@ -92,13 +92,6 @@ git-crypt unlock
 
 Copy the `.env.sample` file and name the new file `.env.development`
 
-To get the tests running you will need to obtain and set values for the following:
-```bash
-CHECK_FINANCIAL_ELIGIBILITY_HOST=
-LEGAL_FRAMEWORK_API_HOST=
-HMRC_API_HOST=
-```
```

Comment:
This is duplicated [here](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/main/README.md?plain=1#L213-L219).